Makefile

```diff
@@ -124,6 +124,9 @@ test: test-unit test-integration test-readme ## test everything that needs test
 test-all: ## test using tox
 	tox -r
 
+.PHONY: test-repo
+test-repo: lint test-unit test-integration test-readme test-performance test-quality ## test everything
```

Comment:
could we just use `test` to replace `test-unit`, `test-integration` and `test-readme` since it covers them?